frontend/src/metabase/lib/urls/actions.ts

```diff
@@ -1,3 +1,9 @@
+import type { WritebackAction } from "metabase-types/api";
+
+export function action(action: WritebackAction) {
```

Comment:
Note: We'll potentially remove the URL from actions if action URL isn't merged to `master` soon. So this is a way to render public entity names without a link